libs/Modal/create/show/index.tsx

```diff
@@ -0,0 +1,170 @@
+import type {
+  ComponentProps,
+  ComponentType,
+  ElementType,
+  FunctionComponent,
+  ReactNode,
+} from "react";
+import ReactDOM from "react-dom";
+import { createRoot, hydrateRoot } from "react-dom/client";
```

Comment:
안쓰는 import 구문은 빼는게 어떨까?
```suggestion
import { createRoot } from "react-dom/client";
```

Reply:
오우 체크 고마워요!